fiasko_bro/code_validator.py

```diff
@@ -9,6 +10,24 @@ def validate_repo(path_to_repo, path_to_original_repo=None):
     return code_validator.validate(path_to_repo, path_to_original_repo)
 
 
+def py_files_count(path_to_repo):
```

Comment:
This function should be in `file_helpers.py`. I know, I know, it will be a single function in a file, but it's ok. So let's put it there.